rules/CodingStyle/Rector/FuncCall/FirstClassCallableRector.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\Node\VariadicPlaceholder;
+use PHPStan\Analyser\Scope;
+use Rector\Rector\AbstractScopeAwareRector;
+use Rector\ValueObject\PhpVersion;
+use Rector\VersionBonding\Contract\MinPhpVersionInterface;
+use ReflectionException;
+use ReflectionFunction;
+use ReflectionNamedType;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Tests\CodingStyle\Rector\FuncCall\FirstClassCallableRector\FirstClassCallableRectorTest
+ */
+final class FirstClassCallableRector extends AbstractScopeAwareRector implements MinPhpVersionInterface
```

Comment:
Both rules have same name at the moment. It should be unique, e.g. `FunctionFirstClassCallableRector`

Reply:
Done

Reply:
as it is on `FunCall`, I guess `FunCallFirstClassCallableRector` is better, but I guess that's a matter of term and DX

Reply:
Awesome :+1: